redisgears_v8_plugin/src/v8_script_ctx.rs

```diff
@@ -449,6 +464,11 @@ impl LibraryCtxInterface for V8LibraryCtx {
                 )));
             }
         }
+
+        self.script_ctx
```

Comment:
Notice that on line 461 on this file. If we return an error we will not unset the `is_being_loaded_from_rdb` value. Maybe we should create a guard struct for this?